src/styles/_catalog.scss

```diff
@@ -1,22 +1,21 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  justify-content: center;
+  gap: 48px;
+  grid-template-columns: $cardWith;
 
-  max-width: 944px;
-  margin: 0 auto;
+  @each $size, $amount in $media {
```

Comment:
If you would like to use media queris like this I would put this variable in the same file for readability.
**$media:
  "488px" 2,
  "768px" 3,
  "1024px" 4;**

Reply:
okay :) thank you @DorotaLeniecDev for your suggestion.